lib/rucio/core/replica.py

```diff
@@ -3855,7 +3855,7 @@ def get_suspicious_files(rse_expression, available_elsewhere, filter_=None, logg
     if not isinstance(nattempts, int):
         nattempts = 0
     if not isinstance(younger_than, datetime):
-        younger_than = datetime.now() - timedelta(days=10)
+        younger_than = datetime.utcnow() - timedelta(days=younger_than)
```

Comment:
@cserf @dchristidis :eyes: pls

Reply:
Ping

Reply:
This isn’t anything I’ve worked on, but I can provide my thoughts.

I would recommend leaving `timedelta(days=10)` as is. Obviously, it’s not right, but I would argue that this function needs to be reviewed in a separate issue. Personally, I regard the fact that it can be either an `int` or a `datetime.dateime` object as a potential source of headaches.

Reply:
Ok, I created #5847 and #5848